IFC4x3/Properties/d/CargoDeadWeight_2vVxFsGGb3QOAIX9UEyhga/DocProperty.xml

```diff
@@ -1,3 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CargoDeadWeight_2vVxFsGGb3QOAIX9UEyhga" Name="CargoDeadWeight" UniqueId="b97fb3f6-4109-4369-8292-84978ef2baa4" PrimaryDataType="IfcMassMeasure" />
```

Comment:
This should not be deleted and replaces 'DeadWeightTonnage' name was altered to remove implicitly stating the unit in the name.

Reply:
I hadn't noticed this change, but I should have and raised it.  This property is listed twice in the baseline I branched from, one in the d folder and one in c
https://github.com/bSI-InfraRoom/IFC-Specification/tree/64f1fc0e8cdf8aba82d2b99a423ddefae2f5a2ba/IFC4x3/Properties/d/CargoDeadWeight_2vVxFsGGb3QOAIX9UEyhga

Reply:
my i see it is fine now!